mssql-ballerina/types.bal

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# MsSQL Geometric Data types.
+
+## Represents Point MsSQL Field
+#
+# + value - Value of parameter passed into the SQL statement
+public class PointValue {
+    public Point | string? value;
+    public isolated function init(Point | string? value = ()) {
+        self.value =value;
+    }
+}
```

Comment:
```suggestion
public distinct class PointValue {
    *sql:TypedValue;
    public Point | string? value;
    
    public isolated function init(Point | string? value = ()) {
        self.value =value;
    }
}
```

Reply:
check and correct all classes. should be distinct and inherit from base object

Reply:
Fixed with [`3941059`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/39410591da9b800c317aa39bad08ab3eeeb71dc8)